upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We have a very extensive `UPGRADE.md file`_ in our repository, which contains almost every breaking change we made
+during the development of Sulu 2.0. However, this is a huge list and most projects won't be affected by the majority of
+these breaking changes. Therefore we are going through the most important changes in this guide, and by following these
```

Comment:
Therefore this guide provides a list of the most important changes. By following the steps of this guide you should be able to update most sulu 1.6 projects to sulu 2.0

Reply:
I'd like to keep the part with overriding, to make more clear when this guide will work.